lib/brexit_checker/results_audiences.rb

```diff
@@ -0,0 +1,56 @@
+class BrexitChecker::ResultsAudiences
+  class << self
+    def populate_business_groups(audience_actions, selected_criteria)
+      return [] if audience_actions.blank? || selected_criteria.blank?
```

Comment:
a different type (array) is returned compared to the hash below

Reply:
does it? it should return a mapped version of `all_possible_groups` which is an array?
If it's not possible to do the work here, i want it to return an empty array. Isn't this right?

Reply:
Ah! i see what you mean, sorry was looking at citizens not business